packages/@react-aria/color/docs/useColorArea.mdx

```diff
@@ -0,0 +1,586 @@
+color channel using a separate control, like a color slider using the [useColorSlider](useColorSlider.html) hook or a color wheel using the
+[useColorWheel](useColorWheel.html) hook.
+
+```tsx example
+import {parseColor} from '@react-stately/color';
+
+function Example() {
+  let [
+    color,
+    setColor
+  ] = React.useState(parseColor('hsb(219, 58%, 93%)'));
+  let [
+    xChannel,
+    yChannel,
+    zChannel
+  ] = color.getColorChannels();
+  return (
+    <>
+      <label id="hsb-label-id-1">x: {color.getChannelName(xChannel, 'en-US')}, y: {color.getChannelName(yChannel, 'en-US')}</label>
+      <ColorArea
+        aria-labelledby="hsb-label-id-1"
+        value={color}
+        onChange={setColor}
+        xChannel={xChannel}
+        yChannel={yChannel} />
+      <label>
+        {color.getChannelName(zChannel, 'en-US')}<br/>
+        <input
+          type="range"
+          min={color.getChannelRange(zChannel).minValue}
+          max={color.getChannelRange(zChannel).maxValue}
+          value={color.getChannelValue(zChannel)}
+          onChange={e => setColor(color.withChannelValue(zChannel, e.target.value))}
+          style={{
+            width: 192
+          }} />
+      </label>
+      <p>Current color value: {color.toString('hsb')}</p>
+    </>
+  );
+}
+```
+
+### onChangeEnd
+
+The `onChangeEnd` prop can be used to handle when a user stops dragging the color area, whereas the `onChange`
+prop is called as the user drags.
+
+```tsx example
+function Example() {
+  let [color, setColor] = React.useState(parseColor('hsl(0, 100%, 50%)'));
+  let [hueChannel, saturationChannel, lightnessChannel] = color.getColorChannels();
+  return (
+    <>
+      <label id="slh-label-id-2">x: {color.getChannelName(saturationChannel, 'en-US')}, y: {color.getChannelName(lightnessChannel, 'en-US')}</label>
+      <ColorArea
+        aria-labelledby="slh-label-id-2"
+        value={color}
+        onChange={setColor}
+        onChangeEnd={setColor}
+        xChannel={saturationChannel}
+        yChannel={lightnessChannel} />
+      <label>
+        {color.getChannelName(hueChannel, 'en-US')}
+        <br />
+        <input
+          type="range"
+          min={color.getChannelRange(hueChannel).minValue}
+          max={color.getChannelRange(hueChannel).maxValue}
+          value={color.getChannelValue(hueChannel)}
+          onChange={(e) =>
+            setColor(color.withChannelValue(hueChannel, e.target.value))}
+          style={{
+            width: 192
+          }}
+        />
+      </label>
+      <p>Current color value: {color.toString('hsl')}</p>
+    </>
+  );
+}
+```
+
+### xChannel and yChannel
+
+The color channel for each axis of a color area can be specified using the `xChannel` and `yChannel` props.
+An array of channel names for a color can be returned using the `color.getColorChannels` method.
+To get a localized channel name to use as a label, you can use the `color.getChannelName` method.
+
+#### RGB
```

Comment:
omg, this example is really cool

Reply:
All the ColorAreas are controlled, it's like one of those psychedelic projections for a 60's rock show.